### DISCUSSION TITLE
Is it possible to catch IMEI as well?

### DISCUSSION THREAD

**Discussion by Maxisldp** [Oros42/IMSI-catcher#120] [Q&A]
hello Oros42, 

first of all, a big thanks to you for this wonderful IMSI-catcher tools. it is really cool!! 
i was wondering is it possible for RTL-SDR to capture IMEI as well? 

your reply will be much appreciated!!
Thanks in advance

**Comment by Oros42** [comment]
Hello and thank you.
Sorry but it's not possible because IMEI is not sent.
